src/Raven.Studio/typescript/models/resources/creation/databaseCreationModel.ts

```diff
@@ -11,7 +11,11 @@ import storageKeyProvider = require("common/storage/storageKeyProvider");
 import setupEncryptionKey = require("viewmodels/resources/setupEncryptionKey");
 import licenseModel = require("models/auth/licenseModel");
 import getCloudBackupCredentialsFromLinkCommand = require("commands/resources/getCloudBackupCredentialsFromLinkCommand");
-import cloudBackupCredentials = require("models/resources/creation/cloudBackupCredentials");
+import { localServerCredentials, 
```

Comment:
use require - this type of import won't work in relase mode IIRC.

Reply:
ok